src/Symfony/Component/Security/composer.json

```diff
@@ -48,6 +48,7 @@
         "symfony/expression-language": "For using the expression voter",
         "ircmaxell/password-compat": "For using the BCrypt password encoder in PHP <5.5"
     },
+    "minimum-stability": "dev",
```

Comment:
Is this really required?

Can you add a suggests for the LDAP in here? 
http://php.net/manual/en/ldap.requirements.php

Maybe it's an idea to throw an exception in the SecurityExtension if LDAP is configured but not installed on the system?